src/frontend/src/app/components/layout/header/header.component.ts

```diff
@@ -8,6 +8,8 @@ import { Subject } from 'rxjs';
 import { takeUntil } from 'rxjs/operators';
 import { SearchTermChangeService } from 'src/app/services/search-term-change.service';
 import { trigger, style, animate, transition } from '@angular/animations';
+import { environment } from 'src/environments/environment.prod';
```

Comment:
You'll want to import the dev environment file. When we build the prod version, it gets replaced with the correct prod paths!